app/assets/stylesheets/components/sub_nav.scss

```diff
@@ -10,21 +9,36 @@
     padding-bottom: 10px;
   }
 
+  .nav-links {
+    display: flex;
+    justify-content: space-around;
+
+    @media (min-width: 640px) {
+      justify-content: flex-end;
+    }
+  }
+
   nav {
     text-align: right;
+    margin-top: 4px;
```

Comment:
I am very dissappointed with this margin top. We previously used flexbox to ensure that the left column and right column were both 100% height. Unfortunately we can't use `display: flex` anymore as it breaks the responsive `-from-desktop` classes that Gov.uk design system provides.

This 4px makes sure the left column and right column are the same height.